openfisca_france_local/departements/eure_et_loir/repas_foyer.py

```diff
@@ -0,0 +1,92 @@
+                 "https://github.com/openfisca/openfisca-france-local/wiki/files/departements/eure-et-loir/RDAS_valide__decembre_2019.pdf"
+                 ]
+    documentation = """
+                        Les personnes en situation de handicap peuvent bénéficier d’une prise en charge des frais de repas servis par des foyers restaurant créés par les communes, les CCAS ou les CIAS habilités à l’aide sociale. 
+                        La participation financière du bénéficiaire est déterminée en fonction de ses ressources et du prix du repas.
+                        Cette aide n’est pas cumulable avec l’Allocation compensatrice pour tierce personne (ACTP), la Majoration Tierce Personne (MTP versée par la CPAM) et les prestations d’aide-ménagère servies par les caisses de retraite
+                    """
+
+    def formula(individu, period, parameters):
+        taux_incapacite = individu('taux_incapacite', period)
+        restriction_substantielle_durable = individu('aah_restriction_substantielle_durable_acces_emploi', period)
+        age = individu('age', period)
+        ressortissant_eee = individu('ressortissant_eee', period)
+
+        repas_foyer_parameters = parameters(
+            period).departements.eure_et_loir.repas_foyer.repas_foyer
+
+        individual_resource_names = {
+            'aah',
+            'salaire_imposable',
+            'retraite_imposable',
+            'pensions_invalidite',
+            'revenus_stage_formation_pro'
+        }
+        ressources_famille = {'aspa'}
+        ressources_annuelles = {'retraite_complementaire_artisan_commercant',
+                                'retraite_complementaire_profession_liberale'
+                                }
+        individu_resources_month = sum(
+            sum([individu(resource, period.last_month) for resource in individual_resource_names]),
+            sum([individu.famille(resource, period.last_month) for resource in ressources_famille]))
+        individu_resources = sum(individu_resources_month, sum(
+            [individu(resource, period, options=[DIVIDE]) for resource in ressources_annuelles]))
+
+        condition_residence = individu.menage('eure_et_loir_eligibilite_residence', period)
+        condition_age = age <= repas_foyer_parameters.age_minimal_personne_handicap
+        condition_nationalite = ressortissant_eee + individu('titre_sejour', period) + individu('refugie',period) + individu('apatride', period)
+        condition_taux_incapacite = ((taux_incapacite >= repas_foyer_parameters.taux_incapacite_superieur)
+                                     + ((taux_incapacite < repas_foyer_parameters.taux_incapacite_maximum_restriction_acces_emploi) * (
+                                         (taux_incapacite > repas_foyer_parameters.taux_incapacite_minimum_restriction_acces_emploi) * restriction_substantielle_durable)))
+        condition_ressources = individu_resources < individu.famille('aspa', period)
```

Comment:
De même, les ressources ne doivent pas dépasser l'ASPA : 
```suggestion
        condition_ressources = individu_resources <= individu.famille('aspa', period)
```